djstripe/models.py

```diff
@@ -978,28 +1017,24 @@ def __str__(self):
         return "<{name}, stripe_id={stripe_id}>".format(name=smart_text(self.name), stripe_id=self.stripe_id)
 
     @classmethod
-    def create(cls, metadata={}, **kwargs):
+    def create(cls, metadata=None, **kwargs):
```

Comment:
Also, it would be fine to just remove metadata from the signature. There are cases when devs might want to set metadata to none, as the [api docs](https://stripe.com/docs/api#create_plan) state:
> [metadata] can be unset by updating the value to None and then saving.